### DISCUSSION TITLE
How to create an border router on esp thread border with AP

### DISCUSSION THREAD

**Discussion by walker2398** [openthread/openthread#11049] [Q&A]
As the title,how can this happen?
I've tried to bind the esp_netif with ap to the backbone router,but it didnt work.
Does it require NAT between sta and ap?

**Comment by chshu** [comment]
The ESP Thread BR in Wi-Fi AP mode is not yet supported. Could you please create an issue at https://github.com/espressif/esp-thread-br/issues and describe your use case? We will follow up with you.

**Reply by walker2398** [reply]
OK then I'll close this discussion thread.
